perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -359,11 +380,15 @@ def Prepare(benchmark_spec):
   # Choose a disk or file name and optionally fill it
   disk = vm.scratch_disks[0]
 
-  if FLAGS.against_device or FLAGS.prefill_device:
+  if FillTarget():
     logging.info('Fill device %s on %s', disk.GetDevicePath(), vm)
-    FillDevice(vm, disk, FLAGS.device_fill_size)
+    FillDevice(vm, disk, FLAGS.fio_fill_size)
 
-  if not FLAGS.against_device and FLAGS.prefill_device:
+  # We only need to format and mount if the target mode is against
+  # file with fill because 1) if we're running against the device, we
+  # don't want it mounted and 2) if we're running against a file
+  # without fill, it was never unmounted (see GetConfig()).
+  if FLAGS.fio_target_mode == AGAINST_DEVICE_WITH_FILL_MODE:
```

Comment:
I thought I was running the tests, but I discovered just now that none of tests tests were actually being run when I did `hooks/check-everything`. I can explain why tomorrow, but I have fixed it, validated that tests are actually run using a test designed to fail, and fixed the errors.